README.md

```diff
@@ -1,36 +1,43 @@
-# Quark
+# Cosmos SDK
 
-Quark is an [ABCI application](https://github.com/tendermint/abci) designed to
-be used with the [Tendermint consensus engine](https://tendermint.com/) to form
-a Proof-of-Stake cryptocurrency.  It also provides a general purpose framework
+![banner](cosmos-sdk-image.png)
+
+[![version](https://img.shields.io/github/tag/cosmos/cosmos-sdk.svg)](https://github.com/cosmos/cosmos-sdk/releases/latest)
+[![API Reference](
+https://camo.githubusercontent.com/915b7be44ada53c290eb157634330494ebe3e30a/68747470733a2f2f676f646f632e6f72672f6769746875622e636f6d2f676f6c616e672f6764646f3f7374617475732e737667
```

Comment:
@zramsay we don't need to hardcode this svg here, we can just use 
https://godoc.org/github.com/cosmos/cosmos-sdk?status.svg
which when set there shows up as
[![API Reference](https://godoc.org/github.com/cosmos/cosmos-sdk?status.svg)](https://godoc.org/github.com/cosmos/cosmos-sdk)